frontend/src/components/search/searchbar/SearchBarView.jsx

```diff
@@ -0,0 +1,43 @@
+import TextField from '@mui/material/TextField'
+import InputAdornment from '@mui/material/InputAdornment'
+import SearchIcon from '@mui/icons-material/Search'
+
+const SearchBarView = ({ data }) => {
+  console.log(data)
+  let text = ''
+  return (
+    <TextField
+      variant="filled"
+      placeholder="Search Updog"
+      autoComplete="off"
+      onChange={(e) => {
```

Comment:
Maybe make it as an individual method instead of inline method, it will be better for later modification

Reply:
The entire project uses lambda functions so I also inline methods to stay consistent across the project. Does anyone else suggest changing this one file to have individual methods?

Reply:
Instead of using let text as a variable, what about using useState?

Reply:
Yeah I also think making `text` a useState would be better. As for the inline functions I think leaving them in should be fine for now for the sake of consistency.

Reply:
Thanks for the suggestions! I've changed it to useState now.